exporter/elasticsearchexporter/model.go

```diff
@@ -57,6 +57,13 @@ var resourceAttrsConversionMap = map[string]string{
 	semconv.AttributeK8SNodeName:            "kubernetes.node.name",
 	semconv.AttributeK8SPodName:             "kubernetes.pod.name",
 	semconv.AttributeK8SPodUID:              "kubernetes.pod.uid",
+	semconv.AttributeK8SJobName:             "kubernetes.job.name",
+	semconv.AttributeK8SCronJobName:         "kubernetes.cronjob.name",
+	semconv.AttributeK8SStatefulSetName:     "kubernetes.statefulset.name",
+	semconv.AttributeK8SReplicaSetName:      "kubernetes.replicaset.name",
+	semconv.AttributeK8SDaemonSetName:       "kubernetes.daemonset.name",
+	semconv.AttributeK8SClusterName:         "kubernetes.cluster.name",
```

Comment:
@carsonip there is no such field as `kubernetes.cluster.name`, it should be removed

Reply:
TODO to self: confirm if it is needed, as it is not in https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-resource-to-ecs%40mappings.yaml

Reply:
Should it maybe point to `orchestrator.cluster.name`? 

From https://github.com/elastic/integrations/blob/main/packages/kubernetes/data_stream/container_logs/fields/ecs.yml#L22 . As @tetianakravchenko pointed out over slack - those fields are covered by ECS under `orchestrator.*` and not under `kubernetes.*`. 

That's the reason why it's not in  https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-resource-to-ecs%40mappings.yaml.

Reply:
Updated